x-pack/plugins/observability/public/components/app/section/alerts/index.tsx

```diff
@@ -49,92 +51,91 @@ export function AlertsSection({ alerts }: Props) {
   }));
 
   return (
-    <SectionContainer
```

Comment:
@katefarrar You should remove unused import above `import { SectionContainer } from '..';`